examples/offline_inference/mlpspeculator.py

```diff
@@ -16,11 +23,15 @@ def time_generation(llm: LLM, prompts: list[str],
     start = time.time()
     outputs = llm.generate(prompts, sampling_params)
     end = time.time()
-    print((end - start) / sum([len(o.outputs[0].token_ids) for o in outputs]))
+    print("-" * 50)
+    print(title)
+    print("time: ",
+          (end - start) / sum([len(o.outputs[0].token_ids) for o in outputs]))
```

Comment:
```suggestion
          (end - start) / sum(len(o.outputs[0].token_ids) for o in outputs))
```

No need to iterate twice